ads/google/doubleclick.js

```diff
@@ -265,3 +245,54 @@ function doubleClickWithGlade(global, data, gladeExperiment) {
 function getCorrelator(global) {
   return makeCorrelator(global.context.clientId, global.context.pageViewId);
 }
+
+function centerAd() {
+  const container = global.document.querySelector('#c');
+  setStyles(dev().assertElement(container), {
+    top: '50%',
+    left: '50%',
+    bottom: '',
+    right: '',
+    transform: 'translate(-50%, -50%)',
+  });
+}
+
+/**
+ * @param {Object} data
+ * @return {!string}
+ */
+export function getUrl(data) {
+  const fileExperimentConfig = {
+    21060540: 'gpt_sf_a.js',
+    21060541: 'gpt_sf_b.js',
+  };
+  // Note that reduce will return the first item that matches but it is
+  // expected that only one of the experiment ids will be present.
+  let expFilename;
+  (data['experimentId'] || '').split(',')
+  .forEach(val => expFilename = expFilename || fileExperimentConfig[val]);
```

Comment:
Hmm I did run gulp lint --fix on this. Does it miss some things? I can format it whatever way seems sensible to you.